bin/dump_data.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Dump component data
+# Run this script from the root of the repository
+# dump_data.sh for all apps or dump_data.sh zaken documenten ... to specify specific apps
```

Comment:
Could you add a docs page that explains this command and refers to this script in github? I think it would also be good to mention how this dumped data can be loaded